gradle-with-plugin/build.gradle

```diff
@@ -134,5 +130,5 @@ dockerCompose {
 }
 
 wrapper {
-    gradleVersion = "5.4.1"
+    gradleVersion = "7.1"
```

Comment:
I do not see an updated `gradle-wrapper.properties` or a new `gradle-wrapper.jar`.

Did you forget to run `./gradlew wrapper` (twice) to actually download the new version?

Reply:
Ah, I missed that. We can remove the `wrapper{}` part. You can upgrade gradle: `gradlew wrapper --gradle-version=7.1`